ui/core/src/views/Workflow.vue

```diff
@@ -1723,16 +1723,16 @@ function condition(){
                     {
                       node: {
                         color: {
-                          background: '#56ec00',
-                          highlight: { background: '#56ec00' },
-                          hover: { background: '#56ec00' }
+                          background: '#f5e203',
```

Comment:
Nice :yellow_heart: 
In the future, we could add a variable here for letting the user choose a color.